### DISCUSSION TITLE
Items with multiple locations

### DISCUSSION THREAD

**Discussion by WesternLibRSC** [CollectionBuilder/collectionbuilder.github.io#97] [Q&A]
I'm working on a library of Milton images from our archives. We've been using the location/lat/long columns in the metadata for indicating publisher. Some works have two publishers, for example: J.&J. Boydel. & G. Nicol, Shakespeare Gallery. J&J is located at Pall Mall and G. Nicol at N.90 Cheapside. So far I've listed them together as the publisher, and put the location as Pall Mall and the lat/long coordinates for that. But I'm wondering how I might list both locations in the spreadsheet.

Should I create second location/lat/long columns and put N.90 Cheapside in it's own column, and if I do, do the column titles need to differ from the first (location, location2). 

My other option is to just tag it broadly as London UK and not get specific. People browsing the collection will still see the publisher information listed with the item in the end. But it would be nice to have both places with coordinates.

Thanks,
Arielle

**Comment by evanwill** [comment]
@WesternLibRSC so are you wanting the item to show up in two different pins on the map? (this isn't possible out of the box, so would involve customizing the map code--however, having an item twice on the map might be confusing from a user perspective)

Or are you just wanting multiple publishers with their own lat/long to be visible on the Item page? (this is possible by just setting up your metadata to have separate lat/long columns with different names)

If you are interested in a map, one customization option that might make sense is to add the publishers as entities on the map, with each publisher being connected to one or more items.  You would set up a new spreadsheet of publishers (name, location, lat, long, etc). Then each of your image items could reference one or more of those publishers in its publisher column. The map feature could be customized to use the publishers rather than image items. 

I think it is a question of what makes sense for users browsing the collection and how useful it will be to browse via map for this content. 

Let us know what you are thinking!

**Reply by cgrillo98** [reply]
Hi! I have a similar case. In my project, each item is composed of sources from different countries. @evanwill, could you explain your solution a bit more? Do you have an example you could share?

**Reply by evanwill** [reply]
@cgrillo98 I can't really think of any examples that implement something like described in the original question.
One sort of related is here with letters, https://cdil.lib.uidaho.edu/mancini/locations.html
the map shows where letters are set from. A separate map shows locations mentions with links to all the related letters.
I have seen a few where the sender and recipient of a letter are both plotted on the same map. 

For your use case, are you imagining plotting the locations of each source for your Item? So each Item might have several related lat longs? It might work to structure your Items like a "compound_object" were the sources are each described with their own row in your spreadsheet, each has it's own lat+long. Those would appear on the map. You would create your own custom item display_template, where you could pull in information about the source rows, and display the main Item record how ever you wanted. 
Which collection are you working on?

**Reply by cgrillo98** [reply]
Very interesting project!
Well, my project is a bit more experimental. We’re creating a collection of animations based on a museum collection in Bogotá. We’re mapping the origins of the original pieces featured in each animation, but some animations are remixes combining multiple pieces, and that’s where we need to assign two or more locations to a single item. You can check it out here: https://cdigitaljaveriana.github.io/museogif/browse.html
The custom item display might be a good option for us 🤔

**Reply by evanwill** [reply]
@cgrillo98 I didn't realize your project was built on CB-GH--> customizing the Item pages is a bit more complicated there--still possible, but requires more messing around with ugly javascript. Let me take a look
